trieste/acquisition/rule.py

```diff
@@ -1068,3 +1082,173 @@ def state_func(
             return state_, points
 
         return state_func
+
+
+class BatchHypervolumeSharpeRatioIndicator(
+    AcquisitionRule[TensorType, SearchSpaceType, ProbabilisticModelType]
+):
+    """Implements the Batch Hypervolume Sharpe-ratio indicator acquisition
+    rule, designed for large batches"""
```

Comment:
You should probably include a citation here (see other docstrings for examples, or ask).